src/main/java/za/ac/cput/factory/RoleFactory.java

```diff
@@ -0,0 +1,23 @@
+package za.ac.cput.factory;
+
+/*RoleFactory.java
+  Factory for Role
+  Author: Zintle Zothe (216130565)
+  Date: 07 April 2022
+ */
+
+import za.ac.cput.entity.Role;
+import za.ac.cput.helper.GenerateID;
+
+
+public class RoleFactory {
+    public static Role createRole(String roleId, String roleName){
```

Comment:
Because the ID will be auto generated, maybe remove the String roleID parameter from the createRole() method.

Reply:
```suggestion
    public static Role createRole(String roleName){
```